client/components/swipeable/docs/example.jsx

```diff
@@ -0,0 +1,28 @@
+import { PureComponent } from 'react';
+import Swipeable from 'calypso/components/swipeable';
+
+export default class extends PureComponent {
+	static displayName = 'Swipeable';
+
+	constructor( props ) {
+		super( props );
+		this.state = { currentPage: 0 };
+	}
```

Comment:
If we decided to use a class component, `state` could be declared as a class instance property, and we wouldn't need to have a constructor:

```suggestion
	state = {
		currentPage: 0,
	};
```